ksqldb-execution/src/main/java/io/confluent/ksql/execution/plan/SourceStep.java

```diff
@@ -32,20 +33,23 @@
   final Formats formats;
   final Optional<TimestampColumn> timestampColumn;
   final LogicalSchema sourceSchema;
+  final int versionNumber;
```

Comment:
Can we rename this to `pseudocolumnVersion`? The current name is misleading since it makes it sound like we're versioning the step itself, but really we're just versioning the pseudocolumns.

Reply:
Ok, I'll wait on his feedback before merging. I've made the other changes locally.